src/util-affinity.c

```diff
@@ -280,7 +281,10 @@ void AffinitySetupLoadFromConfig()
 
         node = ConfNodeLookupChild(affinity->head.tqh_first, "threads");
         if (node != NULL) {
-            taf->nb_threads = atoi(node->val);
+            if (StringParseInt32(&taf->nb_threads, 10, 0, (const char *)node->val) < 0) {
```

Comment:
Can `nb_threads` be negative ?
(if not, I am not sure that this should be part of this PR)

Reply:
Seems like ideally it should not be. But it is read from the conf so maybe its to keep a better check?

Reply:
I would make it unsigned

Reply:
I would make it unsigned

Reply:
Suggest `unsigned` handling too.